sunpy/net/dataretriever/tests/test_farside.py

```diff
@@ -0,0 +1,59 @@
+#This module was developed with funding provided by
+#the Google Summer of Code 2016.
+import datetime
+import pytest
+
+from astropy import units as u
+from sunpy.time.timerange import TimeRange
+from sunpy.net.vso.attrs import Time,Instrument
+from sunpy.net.dataretriever.client import QueryResponse
+from sunpy.net.dataretriever.downloader_factory import UnifiedResponse
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+import sunpy.net.dataretriever.sources.gong as gong
+
+FClient = gong.FARSIDEClient()
+
+@pytest.mark.online
+@pytest.mark.parametrize("timerange, url_start, url_end",
+                         [(TimeRange('2014/2/1', '2014/2/5'),
+                           'http://farside.nso.edu/oQR/fqo/201402/mrfqo140201/mrfqo140201t0000.fits',
+                           'http://farside.nso.edu/oQR/fqo/201402/mrfqo140204/mrfqo140204t1200.fits')])
+def test_get_url_for_timerange(timerange, url_start, url_end):
+    urls = FClient._get_url_for_timerange(timerange)
+    assert isinstance(urls, list)
+    assert urls[0] == url_start
+    assert urls[-1] == url_end
+
+trange = Time('2015/12/28', '2015/12/30')
```

Comment:
why `trange` is outside the test? is it used somwhere else? if it's global then put it in capitals.